resources/wikia/modules/flowTracking.js

```diff
@@ -0,0 +1,57 @@
+define('wikia.flowTracking', ['wikia.tracker', 'wikia.window', 'mw', 'jquery'], function (tracker, w, mw, $) {
+
+	var track = tracker.buildTrackingFunction({
+			category: 'flow-tracking',
+			eventName: 'flowtracking',
+			trackingMethod: 'analytics'
+
+		}),
+		userAgent = w.navigator.userAgent;
+
+	/**
+	 * Track edit flow beginning.
+	 *
+	 * @param flow name of the flow
+	 * @param extraParams additional parameters to track
+	 */
+	function beginFlow(flow, extraParams) {
+		var params = prepareParams(tracker.ACTIONS.FLOW_START, flow, extraParams);
+		track(params);
+	}
+
+	/**
+	 * Track intermediary flow step
+	 *
+	 * @param flow name of the flow
+	 * @param extraParams additional parameters to track
+	 */
+	function trackFlowStep(flow, extraParams) {
+		var params = {};
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "params" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)